vm-setup/roles/common/defaults/main.yml

```diff
@@ -13,6 +13,7 @@ default_memory: 8192
 default_vcpu: 2
 num_nodes: 2
 extradisks: false
+tpm_emulator: false
```

Comment:
So here, as @fmuyassarov says we could instead do e.g:

```
tpm_emulator: "{{ lookup('env', 'VM_TPM_EMULATOR')|default(false, true) }}"
```

Reply:
Oh, that indeed makes much more sense, thanks!